### DISCUSSION TITLE
Any progress made on G6 restarts and bypassing 100 days

### DISCUSSION THREAD

**Discussion by getut** [Navid200/xDrip#113] [Dexcom]
Sorry if this is in another thread, but I haven't found it. I know its been a year or more since I asked last, but is any progress being made cracking nut and getting the G6's to restart/reset and be able to go past 100 days on a transmitter? I'm holding on to all my old transmitters in hopes they will work again some day and maybe even replace batteries in them.

**Comment by Navid200** [comment]
There is a difference between restart and reset.
These are the restart directions:  https://github.com/Navid200/xDrip/wiki/Restart-G6-Sensor

There is no way to hard reset a Firefly G6 as is.

There are individuals who have modified G6.  But, this has nothing to do with xDrip.

**Reply by getut** [reply]
Are there any links to info about modification? The last time I asked, someone said they were still poking at ways to get the G6 to reset via xdrip. But if modification is the only way, are there links?

**Comment by Navid200** [comment]
Google g6 firefly modification
